test/simple/test-tls-destroy-socket.js

```diff
@@ -0,0 +1,55 @@
+if (!process.versions.openssl) {
+  console.error('Skipping because node compiled without OpenSSL.');
+  process.exit(0);
+}
+
+var common = require('../common');
+var assert = require('assert');
+var tls = require('tls');
+var fs = require('fs');
+
+var options = {
+  key: fs.readFileSync(common.fixturesDir + '/keys/agent2-key.pem'),
+  cert: fs.readFileSync(common.fixturesDir + '/keys/agent2-cert.pem')
+};
+
+var normalSock = null;
+var hasError = false;
+var tlsCloseCount = 0;
+var sockCloseCount = 0;
+
+var server = tls.createServer(options, function(secureSock) {
+  secureSock.on('error', function() {
+    hasError = true;
+  });
+  secureSock.on('close', function() {
+    tlsCloseCount ++;
```

Comment:
mmm, didn't the linter complain about the space between `++` and `tlsCloseCount`?